### DISCUSSION TITLE
19-digit numeric string loses precision when converted to number

### DISCUSSION THREAD

**Discussion by yesgs** [vectordotdev/vector#21492] [Q&A] [answered]
to_int("1844928628568102120") = 1844928628568102100

**Comment by jszwedko** [comment, marked as answer]
I don't see this when running `vector vrl`:

```
$ to_int!("1844928628568102120")
1844928628568102120
```

However, I do see it in the VRL playground. The difference is likely to be the WASM target that the playground uses.
